modules/storage-api/src/test/java/org/apache/ignite/internal/storage/chm/TestMvTableStorage.java

```diff
@@ -65,6 +64,11 @@ public CompletableFuture<?> destroyPartition(int partitionId) throws StorageExce
         return CompletableFuture.completedFuture(null);
     }
 
+    @Override
```

Comment:
```suggestion
    /** {@inheritDoc} */
    @Override
```

Reply:
Again, should I break consistency? Let's add it to other methods then